web/web/src/lib/utils/initial.js

```diff
@@ -48,6 +48,34 @@ export const messagingProviders = [
 ]
 
 export const providers = [
+  {
+    label: 'Apache Doris',
+    value: 'jdbc-doris',
+    defaultProps: [
+      {
+        key: 'jdbc-driver',
+        value: '',
+        required: true,
+        description: 'e.g. com.mysql.jdbc.Driver'
+      },
+      {
+        key: 'jdbc-url',
+        value: '',
```

Comment:
Why don't we put `jdbc:mysql://localhost:9030` as the default value?

Reply:
I think it's because, without a default value, the user becomes more aware of the value they are inputting.